config/config.go

```diff
@@ -70,9 +75,22 @@ func New(cfg *config.C) (*Config, error) {
 
 	if c.Benchmark != "" {
 		if !isSupportedBenchmark(c.Benchmark) {
-			return c, ErrBenchmarkNotSupported
+			return c, launcher.NewUnhealthyError(fmt.Sprintf("benchmark '%s' is not supported", c.Benchmark))
 		}
 	}
+
+	switch c.CloudConfig.AwsAccountType {
+	case "":
+	case SingleAccount:
+	case OrganizationAccount:
+		fallthrough // TODO: remove when implemented
+	default:
+		return nil, launcher.NewUnhealthyError(fmt.Sprintf(
+			"aws.account_type '%s' is not supported",
+			c.CloudConfig.AwsAccountType,
+		))
+	}
```

Comment:
It feels to me that it should check that the value appear in a "supported values" array, same as `isSupportedBenchmark` does.
Also, this validation can already support the `OrganizationAccount` and just point it to the single account flow.

Reply:
I've changed this to at least log an error, otherwise the configuration is completely ineffectual in the current code base.